ports/mimxrt/boards/MIMXRT1020_EVK/mpconfigboard.mk

```diff
@@ -16,6 +13,19 @@ else
 JLINK_CONNECTION_SETTINGS =
 endif
 
+CFLAGS += \
+	-DUSDHC_AVAIL=1
+
+SRC_C += \
+	sdcard.c \
+	machine_sdcard.c \
```

Comment:
these two files should not be added by a board; rather they should be included in the build unconditionally and the contents of these files guarded by `#if MICROPY_PY_MACHINE_SDCARD` (ports like stm32 and esp32 use `MICROPY_HW_ENABLE_SDCARD` and arguably these two can both coexist, the first for the `machine.SDCard` class and the second for enabling the low-level C driver)

Reply:
fixed.